tests/UseStatementParserTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\VarDumper\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Yiisoft\VarDumper\UseStatementParser;
+
+class UseStatementParserTest extends TestCase
+{
+    /**
+     * @dataProvider usesProvider
+     */
+    public function testFromFile(string $file, array $expectedUses)
+    {
+        $parser = new UseStatementParser();
+
+        $actualUses = $parser->fromFile($file);
+
+        $this->assertEquals($expectedUses, $actualUses);
+    }
+
+    public function usesProvider(): array
+    {
+        /**
+         * use Yiisoft\Arrays\ArrayHelper;
+         * use Yiisoft\Arrays\ArrayHelper, Yiisoft\Arrays\ArraySorter;
+         * use Yiisoft\{Arrays\ArrayHelper, Arrays\ArrayableTrait};
+         * use Yiisoft\{Arrays\ArrayHelper, Arrays\ArrayableTrait}, Yiisoft\Arrays\ArraySorter;
+         */
+        return $this->saveExamplesToTemporaryFile([
+            [
+                'use Yiisoft\Arrays\ArrayHelper as alias;',
+                ['alias' => '\Yiisoft\Arrays\ArrayHelper'],
+            ],
+            [
+                'use Yiisoft\Arrays\ArrayHelper as Helper, Yiisoft\Arrays\ArraySorter as Sorter;',
+                [
+                    'Helper' => '\Yiisoft\Arrays\ArrayHelper',
+                    'Sorter' => '\Yiisoft\Arrays\ArraySorter',
+                ],
+            ],
+            [
+                'use Yiisoft\Arrays\ArrayHelper;',
+                ['ArrayHelper' => '\Yiisoft\Arrays\ArrayHelper'],
+            ],
+            [
+                'use Yiisoft\Arrays\ArrayHelper, Yiisoft\Arrays\ArraySorter;',
+                [
+                    'ArrayHelper' => '\Yiisoft\Arrays\ArrayHelper',
+                    'ArraySorter' => '\Yiisoft\Arrays\ArraySorter',
+                ],
+            ],
+            [
+                'use Yiisoft\{Arrays\ArrayHelper, Arrays\ArrayableTrait};',
+                [
+                    'ArrayHelper' => '\Yiisoft\Arrays\ArrayHelper',
+                    'ArrayableTrait' => '\Yiisoft\Arrays\ArrayableTrait',
+                ],
+            ],
+        ]);
+    }
+
+    public function saveExamplesToTemporaryFile(array $examples): array
+    {
+        static $handles = [];
```

Comment:
Is it a leftover? Doesn't seem to be used...

Reply:
No it's needed for tests. `usesProvider` provides temporary file that contains code from provider.

Reply:
Need an inline comment then. I bet every person reading the code will be eager to delete these lines.